API_Guidelines.md

```diff
@@ -9,6 +9,7 @@
 	[ [Single Record](#4.2-GET-a-single-record) ]
 	[ [Querying Multiple Records](#4.3-GET-multiple-records-using-a-query)  ]
 	[ [Querying a Single Record](#4.4-GET-a-single-record-using-a-query) ]
+	[ [Querying Using an Array](#4.5-querying-using-an-array) ]
```

Comment:
The decimals in the numbers break these links, and can you update all of these links to be lowercase?